lib/core/datatable/components/datatable/datatable.component.html

```diff
@@ -25,13 +25,7 @@
                  (keyup.enter)="onColumnHeaderClick(col)"
                  role="columnheader"
                  tabindex="0"
-                 [attr.aria-sort]="col.sortable ?
-                    ((isColumnSortActive(col) ?
-                        (isColumnSorted(col, 'asc') ?
-                            'ADF-DATATABLE.ACCESSIBILITY.SORT_ASCENDING' :
-                            'ADF-DATATABLE.ACCESSIBILITY.SORT_DESCENDING') :
-                        'ADF-DATATABLE.ACCESSIBILITY.SORT_NONE') | translate) :
-                    null "
+                 [attr.aria-sort]="col.sortable ? (getAriaSort(col) | translate) : null"
```

Comment:
can be `(getAriaSort(col) | translate)` and the `getAriaSort` checking for sortable and returning null

Reply:
null is used not to set aria-sort attribute for columns that are not sortable